src/alerting/utils/history.test.ts

```diff
@@ -0,0 +1,153 @@
+import * as history from 'src/alerting/utils/history'
+import {fromFlux, Table} from '@influxdata/giraffe'
+import {range, uniq} from 'lodash'
+import {mocked} from 'ts-jest/utils'
+import {runQuery} from 'src/shared/apis/query'
+
+jest.mock('@influxdata/giraffe', () => ({
+  fromFlux: jest.fn(),
+  // todo: test will fails on binaryPrefixFormatter not a function in src/shared/copy/notifications.ts:22:24 this mock can be removed after this will be fixed
```

Comment:
What exactly is the issue with `binaryPrefixFormatter`? Is this something we need to fix in Giraffe? Do we have an issue for it?

Reply:
[Issue](https://github.com/influxdata/giraffe/issues/614) created. I tried to investigate the problem but doesn't look like something I could solve (at least from ui side).